frontend/index.html

```diff
@@ -2,18 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/favicon.ico" />
```

Comment:
no need for PUBLIC_URL here as vite handles that nicely
https://vitejs.dev/guide/assets.html
https://vitejs.dev/guide/build.html#public-base-path

Reply:
Did you try with a different baseURL?

Reply:
yep, it works